modules/cnv-labels-nodes-workloadscheduling.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_advanced_vm_management/cnv-dedicated-resources-vm.adoc
+// * cnv/cnv_vm_templates/cnv-dedicated-resources-vm-template.adoc
+
+[id="cnv-labels-nodes-workloadscheduling_{context}"]
+
+= Preparing to enable dedicated resources
+
+Prepare to enable dedicated resources by doing the following.
+
+.Procedure
+* Ensure that the `cpumanager` label is `true` for each node on which you want
+to schedule virtual machines with dedicated resources.
+** Verify that CPU Manager is running on the CPU node whose
+resources you want to dedicate.
+** Set the CPU manager policy that is running on the node to `static`.
+
+NOTE: If the `cpumanager` label is not `true`, you receive a warning message
```

Comment:
Rather than a note, I would suggest this be a para in the intro para of this procedure.
I also think this can be summarised to something like "If the `cpumanager=true` label isn't applied to an available node, the workload can not be scheduled."

Reply:
OK, you had initially suggested a note when we slacked but maybe I misinterpreted.  

Intro to this prep section now reads:

"If the `cpumanager` label isn't applied to an available node, 
the workload can not be scheduled."